docs/k8s/config.md

```diff
@@ -53,6 +57,44 @@ spec:
           readOnly: true
 ```
 
+## Telemetry
+
+Since Imageless Kubernetes uses the Flox CLI to perform certain operations such as activating your environment, Imageless Kubernetes will report the same telemetry by default that the Flox CLI reports.
+This includes information such as:
+
+- Which subcommands were run
+- Which shell was used for the activation (Bash, Zsh, etc)
+- Whether the environment was remote (e.g. stored on FloxHub) or not
+- etc
```

Comment:
nonblocking suggestion: kind of a +1 for what Bill said, but the way this is phrased feels more relevant for the CLI and isn't relevant in Kubernetes (shell is always bash, environment is always remote)

Or below
```
When using the Flox CLI directly you can set `FLOX_DISABLE_METRICS=1` in your environment
```

feels like it belongs somewhere else in the docs